### PR TITLE
accessibility statement configuration completed

### PR DESCRIPTION
**Pull request by thirlz95**

### CODE REVIEW

File: conf/application.conf [hunk 1]
```diff
@@ -176,3 +176,11 @@ microservice {
 webchat {
   id = TT55004894
 }
+
+tamc-frontend {
+    host="http://localhost:9900"
+}
+
+accessibility-statement {
+    url="http://localhost:12346"
```
Comment (matthew-mcnally) [inline conf/application.conf:185]: Port 12346? Is this a dummy port or should this be 9900?

**Review by danewatts — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (matthew-mcnally) [inline conf/application.conf:185]: Port 12346? Is this a dummy port or should this be 9900?

**Review by matthew-mcnally — CHANGES_REQUESTED** [review]

File: conf/application.conf (hunk 1 above)
Comment (thirlz95) [reply conf/application.conf:185]: Port 123456 is the local port used for accessibility-statement service, I'll be adding configuration changes for Prod/staging shortly

**Review by matthew-mcnally — APPROVED** [review]

File: conf/application.conf (hunk 1 above)
Comment (danewatts) [reply conf/application.conf:185]: 12346, but yes - https://github.com/hmrc/accessibility-statement-frontend/blob/master/build.sbt#L52

**Pull request merged**